src/OpenLoco/Interop/Hooks.cpp

```diff
@@ -60,31 +60,28 @@ using namespace OpenLoco;
 FORCE_ALIGN_ARG_POINTER
 static int32_t CDECL audioLoadChannel(int a0, const char* a1, int a2, int a3, int a4)
```

Comment:
are these used at all now? should they be deleted?

Reply:
Yeah i think it's safe to delete them

Reply:
sounds good, i love deleting code